README.md

```diff
@@ -2,37 +2,48 @@
 
 A simple project template using [Melange](https://github.com/melange-re/melange).
 
-## Getting started
+## Quick Start
 
-You will need `npm` installed, as well as the [esy](https://esy.sh) package
```

Comment:
New users no longer need to get `esy` installed because it's now a `devDependency` that gets run through the npm scripts, and is therefore auto-installed when doing the `npm install` step.